R/Previewer.R

```diff
@@ -181,6 +181,21 @@ block_to_html <- function(b) {
   )
 }
 
+#' @keywords internal
```

Comment:
I proposed a markdown or a pure Java Script solution.

Reply:
I don't think using JS is a great idea so of the two proposed options I'd go with markdown.

However, I don't see how it's worth it to add markdown when we already depend on rmarkdown. It works great in the simple example app you provided. But to feed input directly to `markdown::markdownToHTML` would require refactoring a lot of the code base. The add card modal and the report previewer are separate modules and the comment box content right now is stored in a `TextBlock` class. I don't think it would be simple to get all this working.

I'm OK going down this route if that's what is decided but that goes beyond this issue and PR. I think we would need more discussion and more details fleshed out before continuing in this way.